package.json

```diff
@@ -1,23 +1,33 @@
 {
   "name": "storybook-source-link",
-  "version": "4.0.0",
+  "version": "5.0.0",
```

Comment:
Seems to me like you update versions manually beforehand?

I personally keep `  "version": "0.0.0-semantically-released",` in my addons and let `auto` do its thing.